core/src/repair_service.rs

```diff
@@ -22,6 +22,14 @@ pub const MAX_REPAIR_TRIES: u64 = 128;
 pub const NUM_FORKS_TO_REPAIR: usize = 5;
 pub const MAX_ORPHANS: usize = 5;
 
+pub enum RepairStrategy {
+    RepairRange(RepairSlotRange),
+    Repair {
```

Comment:
The above field is qualified with "Range". Should this one be too? "All" perhaps?